hugolib/node_as_page_test.go

```diff
@@ -385,6 +386,25 @@ title = "Deutsche Hugo"
 	assertFileContent(t, expectedFilePath(ugly, "public", "nn", "sect1", "regular2"), true,
 		"Single Title: Page 02")
 
+	// Page Params
+	pages := sites.findAllPagesByKind(KindPage)
+	for _, p := range pages {
+		if p.Path() == "sect1/regular1.en.md" {
```

Comment:
Also, and I know I have been a sinner in area, adding more test cases to an existing very broad test to test for a very specific thing smells a little. I think we have narrower unit-typish tests for these params conversions -- and if not, we should.